publish/prepare-artifacts.proj

```diff
@@ -137,6 +137,57 @@
       Importance="High" />
```

Comment:
This (and the line above it) are also not doing anything, as the task spits out the vso artifact upload message already. I'd remove since it's uploading the same artifact twice and this can do odd things sometimes.